dffml/df/memory.py

```diff
@@ -1431,12 +1437,11 @@ def __init__(
         ctx_str = (await ctx.handle()).as_string()
         # Create initial events to wait on
         # TODO(dfass) Make ictx.added(ctx) specific to dataflow
-        input_set_enters_network = asyncio.create_task(self.ictx.added(ctx))
-        tasks.add(input_set_enters_network)
         # schedule running of operations with no inputs
         async for task in self.nctx.dispatch_auto_starts(self, ctx):
             tasks.add(task)
-
+        input_set_enters_network = asyncio.create_task(self.ictx.added(ctx))
+        tasks.add(input_set_enters_network)
```

Comment:
Was there a reason for moving this?

Reply:
Ah, I see your Gitter message. That's not good if we don't know why :/

Reply:
Yeah,been trying to figure it out.I was trying some stuff and this swapping was a total fluke.

Reply:
The operations without inputs are dispatched outside the loop of other operations,and since earlier they were added after `input_set_enters_network` ,the network was not getting notified of the outputs from the async gen.
^ Reason for the swap